packages/gatsby/src/redux/save-state.js

```diff
@@ -1,14 +1,21 @@
 const _ = require(`lodash`)
 const report = require(`gatsby-cli/lib/reporter`)
+const { captureEvent } = require(`gatsby-telemetry`)
 const redux = require(`./`)
 
 let saveInProgress = false
 async function saveState() {
   if (saveInProgress) return
   saveInProgress = true
 
+  const startTime = new Date()
+
   try {
     await redux.saveState()
+
+    const duration = (new Date().getTime() - startTime.getTime()) / 1000
```

Comment:
This needs to be an Integer

Reply:
How do we track duration in other places? Ill follow

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime returns the same thing as `Date.now()`

Reply:
used Date.now